### DISCUSSION TITLE
Is there a ST4 Python target?

### DISCUSSION THREAD

**Discussion by Lucretia** [antlr/antlr4#4648] [General]
I'm learning python for a specific binding project and the antlr page says there is a ST4 Python target, but there is only for antlr3, is there an ST4 port? How similar are these?

**Comment by mike-lischke** [comment]
I can offer a port to [TypeScript](https://github.com/mike-lischke/stringtemplate4ts). It's the better language anyway 😉

**Comment by Lucretia** [comment]
JS is a poor language, TS is just a crutch.
